integration-tests/tests/resource-observer/resource-obs.spec.js

```diff
@@ -0,0 +1,50 @@
+/*
+Copyright 2020 Splunk Inc.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+module.exports = {
+  'should report resource loads happening after page load': async function(browser) {
+    browser.globals.clearReceivedSpans();
```

Comment:
Perhaps it's here and I'm not seeing it, but can we get a test that ensures that we don't now double-report resources from the docload sequence?  I.e., there should be only 1 span for an image that is present in the original HTML, not 2.

Reply:
It depends if the image is cached or not. If it is not cached then there should be a span if cached and no network request done then no span. Added two new tests to test this cache/no-cache situation.

Reply:
I just meant that by having a separate observer we run the risk of reporting two spans for the same original performance event, regardless of cache/no-cache.  Adding cache/no-cache tests is great too, but I was asking for something simpler.

Reply:
Added asserts checking that there's only one span for resources loaded during page load.